### PR TITLE
fix(ajax): bug fixes and readability improvements

### PR DESCRIPTION
**Pull request by narzac**
## What I did

1. Fix options expansion
2. Fix removing request interceptors
3. Use 1 hour as the default time to live
4. Check for null when serializing the search params

### CODE REVIEW

File: packages/ajax/src/AjaxClient.js [hunk 1]
```diff
@@ -27,11 +27,11 @@ export class AjaxClient {
       xsrfCookieName: 'XSRF-TOKEN',
       xsrfHeaderName: 'X-XSRF-TOKEN',
       jsonPrefix: '',
+      ...config,
```
Comment (jorenbroekema) [inline packages/ajax/src/AjaxClient.js:30]: Ah good catch

File: packages/ajax/src/AjaxClient.js [hunk 2]
```diff
@@ -123,11 +115,11 @@ export class AjaxClient {
       // In this instance we actually do want to await for each sequence
       // eslint-disable-next-line no-await-in-loop
       const interceptedRequestOrResponse = await intercept(interceptedRequest);
-      if (interceptedRequestOrResponse instanceof Request) {
-        interceptedRequest = interceptedRequestOrResponse;
-      } else {
+      if (interceptedRequestOrResponse instanceof Response) {
```
Comment (jorenbroekema) [inline packages/ajax/src/AjaxClient.js:118]: Lars refactored this code a bit in his PR that I just merged, so please rebase and check whether your change is still required.

**Review by jorenbroekema — APPROVED** [review]
Looks good to me! There will be some merge conflicts for sure with the refactors from https://github.com/ing-bank/lion/pull/1305 but hopefully nothing too annoying.

File: packages/ajax/src/AjaxClient.js (hunk 1 above)
Comment (jorenbroekema) [reply packages/ajax/src/AjaxClient.js:30]: I would see this as a bugfix, if you have time could you add a regression test for this fix? And then we should release it as a patch as well and make clear what was fixed in the changelog, by running `npx run changeset` :)

Comment (narzac) [reply packages/ajax/src/AjaxClient.js:30]: Sure, planning to :)

File: packages/ajax/src/AjaxClient.js (hunk 2 above)
Comment (narzac) [reply packages/ajax/src/AjaxClient.js:118]: Cool, I will rebase

**Comment by narzac** [comment]
I think it is ready to merge

**Review by jorenbroekema — APPROVED** [review]

**Pull request merged**